Page was created by Develop

BoGoBooM opened the pull request:
- [Demo link](https://bogoboom.github.io/Potr_Pots/)

loralevitska reviewed (changes requested):
You did an amazing job! Looks good! Let's do some improvements (check comments)

- after autocomplete the form, change the default styles. Read more about [changing autocomplete styles](https://css-tricks.com/snippets/css/change-autocomplete-styles-webkit-browsers/)
![image](https://github.com/mate-academy/Potr_Pots/assets/70009205/9524be43-6b1b-4a8a-a9f8-e0242dd8b00c)
- fix menu, it doesn't work
- add smooth scroll to page after navigation link click
- fix this view for mobile mode
![image](https://github.com/mate-academy/Potr_Pots/assets/70009205/03ea7e3e-bbcb-4f46-92ef-e3df6c8121b3)
- for mobile mode slider disappear after first click 
![image](https://github.com/mate-academy/Potr_Pots/assets/70009205/5058213c-9030-467a-858d-2c4fe5241eda)

BoGoBooM commented:
Due to a long break from working on javascript, I can't properly implement the slider script, so I decided to leave the best version of it out of all that I had)

anastasiiavorobiova reviewed (changes requested):
Great work! 
<img width="339" alt="Screenshot 2023-11-11 at 18 08 34" src="https://github.com/mate-academy/Potr_Pots/assets/135232001/80802f2c-cd55-4c0c-89d6-41264b383e85">
<img width="916" alt="Screenshot 2023-11-11 at 18 08 51" src="https://github.com/mate-academy/Potr_Pots/assets/135232001/213e453b-ce33-405b-8d9d-1925773a4d1f">
Add a rounded image here 
<img width="1022" alt="Screenshot 2023-11-11 at 18 07 43" src="https://github.com/mate-academy/Potr_Pots/assets/135232001/5d06ea94-29d4-4865-8916-3c605c032828">
Check margins here 
<img width="802" alt="Screenshot 2023-11-11 at 18 05 24" src="https://github.com/mate-academy/Potr_Pots/assets/135232001/981d36a8-7651-4248-a7bf-1343d64278ad">
Consider removing the horizontal scroll 
<img width="696" alt="Screenshot 2023-11-11 at 18 05 01" src="https://github.com/mate-academy/Potr_Pots/assets/135232001/98c03199-4ef4-4732-9a05-543e502f540a">

<img width="1053" alt="Screenshot 2023-11-11 at 18 05 08" src="https://github.com/mate-academy/Potr_Pots/assets/135232001/238ca354-11e5-462d-9f8e-efc4a509e321">
Check the layout for the first section 
<img width="1391" alt="Screenshot 2023-11-11 at 18 04 26" src="https://github.com/mate-academy/Potr_Pots/assets/135232001/636912d3-1fc2-4599-b8fc-61cc3512710a">
Check the font size here 
<img width="777" alt="Screenshot 2023-11-11 at 18 02 34" src="https://github.com/mate-academy/Potr_Pots/assets/135232001/63bab95b-f1ac-4b5a-bc21-1aaee64ecc2c">
<img width="787" alt="Screenshot 2023-11-11 at 18 02 44" src="https://github.com/mate-academy/Potr_Pots/assets/135232001/9da5ec6e-c041-4c2f-bf67-61d7bd352488">
<img width="774" alt="Screenshot 2023-11-11 at 18 02 52" src="https://github.com/mate-academy/Potr_Pots/assets/135232001/a37c04b0-0929-490f-8ae7-eadb836484a2">
<img width="771" alt="Screenshot 2023-11-11 at 18 02 59" src="https://github.com/mate-academy/Potr_Pots/assets/135232001/39deff7d-e093-4148-9f0c-509f5b507aeb">
<img width="1069" alt="Screenshot 2023-11-11 at 18 03 10" src="https://github.com/mate-academy/Potr_Pots/assets/135232001/a6fb8b4e-0a61-41dd-98a9-32a0dfd396e9">
Consider fixing validation errors 
<img width="635" alt="Screenshot 2023-11-11 at 18 01 05" src="https://github.com/mate-academy/Potr_Pots/assets/135232001/f97aa332-6d50-4ee5-8da4-cc5801a2b1ad">
There should be a link to the index page 
<img width="562" alt="Screenshot 2023-11-11 at 18 00 57" src="https://github.com/mate-academy/Potr_Pots/assets/135232001/3021adbc-1dba-4055-91c4-f927087c52d4">
Check the button size 
<img width="362" alt="Screenshot 2023-11-11 at 18 00 18" src="https://github.com/mate-academy/Potr_Pots/assets/135232001/dc6125db-a59f-4e54-a223-2f5eb6e6cdd2">
Add a cursor pointer to the interactive elements

Andriy-Shymkiv reviewed (approved):
add cursor pointer for all buttons:
![image](https://github.com/mate-academy/Potr_Pots/assets/102289654/ffe5ceb6-94c1-4150-bd92-49053ad2f99a)
![image](https://github.com/mate-academy/Potr_Pots/assets/102289654/f350f926-28ba-41ed-aeb2-88a71794b645)
![image](https://github.com/mate-academy/Potr_Pots/assets/102289654/9c4d324e-3f9a-4aae-a630-c5361402b706)

volodymyr-soltys97 reviewed (changes requested):
Good job 👍 
Let's improve your page
1. Make all these fields are required before submit.
Also, when clicking on the button, only the fields should be cleared and should not jump to the top of the page
<img width="424" alt="image" src="https://github.com/mate-academy/Potr_Pots/assets/64750958/80193f21-6285-4166-8670-18035f36a861">

BoGoBooM commented:
I added a script to submit a form without reloading the page, because my script is not working correctly. my form scrolls to the header after submission and I don't know how to fix it.

anastasiia-tilikina reviewed (changes requested):
Good job!

pls watch the [video](https://www.loom.com/share/176f8ce310204928a02d6b5c7a988923?sid=5fdff97a-f700-4a41-8675-d6b004766a22) and fix issues.

loralevitska reviewed (changes requested):
You did an amazing job! Looks good! Let's do some improvements (check comments)

- make all form field required
- fix this part according design 
![image](https://github.com/mate-academy/Potr_Pots/assets/70009205/cc04a6cc-1b33-482e-a770-df5f22211ff8)
mockup 
![image](https://github.com/mate-academy/Potr_Pots/assets/70009205/a10b6a60-7ac0-4d00-856a-2a576cb7ec51)
- would be better to change it to 2023 year 
![image](https://github.com/mate-academy/Potr_Pots/assets/70009205/6c22d770-d51f-4ba0-a888-6314e96e9529)
- remove this line from icon 
![image](https://github.com/mate-academy/Potr_Pots/assets/70009205/9f7155d7-8b50-4ae7-896f-30b178914586)
- fix mobile mode view 
![image](https://github.com/mate-academy/Potr_Pots/assets/70009205/07f7fdcf-006b-4c17-ab8f-2138d5c0b5d2)
- on mobile mode the scroll disabled even if menu isn't open (to reproduce it open and close menu)

BoGoBooM commented:
I'm sorry, I really want to fix all the points you showed me, but I don't see these problems on my computer and I don't know why it happens and how to fix it

l4st1m0za commented:
Hey! Right now your demo link is not working properly. May be you have forgotten to deploy the changes or you have cahnged something so that it hurt the link. I will not make a review for now, so you can fix the link and your home task will be automatically send for the review once more, so don't worry!

BoGoBooM commented:
I replaced 2023 with 2024 in order to make the npm run deploy. Everything should be fine now. The last time I closed the project early, it didn't have time to be published.

anastasiiavorobiova reviewed (changes requested):
Great work! Almost done! A user should be able to scroll the page after closing the menu by clicking on the close button. 
<img width="719" alt="Screenshot 2023-11-26 at 16 11 24" src="https://github.com/mate-academy/Potr_Pots/assets/135232001/8c296f6d-c365-4f6b-b1c8-9c68a0ffec30">
<img width="898" alt="Screenshot 2023-11-26 at 16 11 33" src="https://github.com/mate-academy/Potr_Pots/assets/135232001/f3d0c23b-81e8-4fdc-a45d-74c279c48626">
<img width="861" alt="Screenshot 2023-11-26 at 16 11 42" src="https://github.com/mate-academy/Potr_Pots/assets/135232001/eb4ee1c8-bf90-4962-8721-a7dfb3d6f0b6">
<img width="669" alt="Screenshot 2023-11-26 at 16 11 49" src="https://github.com/mate-academy/Potr_Pots/assets/135232001/e95771da-84b2-4241-9b89-b115ce2f5be6">
<img width="1065" alt="Screenshot 2023-11-26 at 16 12 02" src="https://github.com/mate-academy/Potr_Pots/assets/135232001/2a3002eb-7f56-453a-859e-609b2100d0bc">
Consider fixing validation errors. IDs on the page should be unique

BoGoBooM commented:
I have fixed the menu scripts and hope that you will approve this work

BoGoBooM commented:
because I can't go further.

anastasiia-tilikina reviewed (changes requested):
I'm really sorry about your feelings(

but still we have an issue to be fixed. I belive it will be our last iteration!
You did an incredible job here! 🔥 

pls watch my [video](https://www.loom.com/share/15c20b9467624f4ea1f058909997b3a1?sid=f135a59a-d5f7-4cd6-9498-0527f615f9ba)

volodymyr-soltys97 reviewed (changes requested):
Still not fixed from the previous review, you need to remove the scroll when this menu is open
<img width="1163" alt="image" src="https://github.com/mate-academy/Potr_Pots/assets/64750958/81f3020c-8596-4f19-b3c3-37a9d8294971">

BoGoBooM commented:
finally, I fixed the scrolling in the menu section. The best part is that I spent the whole day trying to fix this error using chatGPT because I don't understand javascript very well, and in the evening I just sat down to think a little and fixed it myself

volodymyr-soltys97 reviewed (approved):
Great work 🔥
